linera-views-derive/src/lib.rs

```diff
@@ -127,11 +127,11 @@ fn generate_view_code(input: ItemStruct, root: bool) -> TokenStream2 {
 
     let increment_counter = if root {
         quote! {
-            linera_views::increment_counter(
-                linera_views::LOAD_VIEW_COUNTER,
-                stringify!(#struct_name),
-                &context.base_key(),
-            );
+            linera_views::LOAD_VIEW_COUNTER.with_label_values(
+                &[
+                    stringify!(#struct_name).into(),
+                    &hex::encode(&context.base_key())
```

Comment:
It probably comes from me using `hex` here in this file, but not sure what I could do differently

Reply:
Figured it out